addons/hr_holidays/models/hr_holidays.py

```diff
@@ -439,6 +439,22 @@ def action_approve(self):
             else:
                 holiday.action_validate()
 
+    @api.model
+    def _prepare_create_by_category(self, holiday, employee):
```

Comment:
why not : `def _prepare_create_by_category(self, employee):` since self is already a holiday?